src/backend/src/services/projects.services.ts

```diff
@@ -949,18 +948,8 @@ export default class ProjectsService {
     });
 
     if (!materialType) throw new NotFoundException('Material Type', materialTypeId);
-    if (materialType.dateDeleted) throw new DeletedException('Material Type', materialTypeId);
 
-    const deletedMaterialType = await prisma.material_Type.update({
-      where: {
-        name: materialTypeId
-      },
-      data: {
-        dateDeleted: new Date()
-      }
-    });
-
-    return deletedMaterialType;
+    await prisma.material_Type.delete({ where: { name: materialTypeId } });
```

Comment:
we still wanna return the deletedMaterialType in the response. This prisma call returns the object it deletes, so you can just return that to the controller and put it in the response